### DISCUSSION TITLE
How to use multiple git accounts in same system ?

### DISCUSSION THREAD

**Discussion by ghost** [EddieHubCommunity/support#3751] [Q&A]
I have two github accounts and I have to use them regularly. How can I configure the global config file so that I don't have to login for every time I make a push.

I tried changing local name and email using `git config user.name "name"` But it didnot work

**Comment by eddiejaoude** [comment]
I have never tried to use multiple git accounts, not sure why someone would want to do this 🤔 

Maybe have a look at GitHub docs https://docs.github.com/en/get-started/getting-started-with-git/setting-your-username-in-git

**Reply by ghost** [reply]
My situation for example.. office account and personal account..

**Comment by Panquesito7** [comment]
This is a very good question. I never thought of this, actually. 👀
I did a quick search on Google and found this. From a quick view, it looks good, but you might want to check it yourself: https://blog.gitguardian.com/8-easy-steps-to-set-up-multiple-git-accounts/

Don't forget to search up for other resources regarding this or ask in communities! 🙂

**Comment by awibox** [comment]
Hello. You can cnahge a git config for a specific repository.
You need to go to the repository folder, then edit the git config for example using nano:
`nano .git/config`
And add the necessary settings there:
`[user]
    name = "Your name"
    email = test@test.com
`
Save settings and that's it
